prov/efa/test/efa_unit_test_mocks.c

```diff
@@ -130,7 +130,7 @@ int efa_mock_ibv_next_poll_use_saved_send_wr_with_mock_status(struct ibv_cq_ex *
 	return efa_mock_use_saved_send_wr(ibv_cqx, mock());
 }
 
-void efa_mock_ibv_end_poll_check_mock(struct ibv_cq_ex *ibvcqx)
+void efa_mock_ibv_end_poll_return_mock(struct ibv_cq_ex *ibvcqx)
```

Comment:
I do not think this new name is right. This function does not return `mock`, it only call `mock`.

Reply:
Will address.